packages/integration-tests/test/transferTest.js

```diff
@@ -221,4 +221,232 @@ describe('Transfer tests', async () => {
       assert.equal(bobUtxos[1].currency, transaction.ETH_CURRENCY)
     })
   })
+
+  describe.skip('ERC20 transfer', async () => {
```

Comment:
Because the setup is too manual at the moment for ERC20 tests, and we want to use these integration tests in CI to validate the platform.
We need to come up with a solution for this. The main issue is adding the erc20 contract address to the fee_spec file. One solution is to use a well known erc20 contract e.g. WETH